config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json

```diff
@@ -28,20 +28,22 @@
         "4.3.0": {
             "status": "deprecated",
             "latest-revision": {
-                "revision": "11",
-                "release-date": "17 November 2022",
-                "cves": "CVE-2022-1304,CVE-2016-3709",
-                "golang": "1.18.6",
-                "ubi": "8.7-923",
+                "revision": "12",
+                "release-date": "09 January 2023",
+                "cves": "CVE-2022-42898",
+                "golang": "1.18.8",
+                "ubi": "8.7-923.1669829893",
                 "new-features": [],
-                "fixes": []
+                "fixes": [
+                    "Fixes CVE-2022-42898 and PSIRT PVR0406812"
+                 ]
             },
             "previous-revision": {
-                "revision": "10",
-                "release-date": "07 November 2022",
-                "cves": "CVE-2022-37434,CVE-2020-35525,CVE-2020-35527,CVE-2022-3515,CVE-2022-2509,CVE-2022-40674",
+                "revision": "11",
+                "release-date": "17 November 2022",
+                "cves": "CVE-2022-42898",
```

Comment:
Fix "cves" key in "previous-revision":
expected value:
     "cves": "CVE-2022-1304,CVE-2016-3709"